nexus/reconfigurator/execution/src/clickhouse.rs

```diff
@@ -181,11 +181,13 @@ pub(crate) async fn deploy_single_node(
         let admin_url = format!("http://{admin_addr}");
         let log = opctx.log.new(slog::o!("admin_url" => admin_url.clone()));
         let client = ClickhouseSingleClient::new(&admin_url, log.clone());
-        client.init_db().await.map(|_| ()).map_err(|e| {
-            anyhow!(
-                "failed to initialize single-node clickhouse database: {e}",
-            )
-        })
+        if let Err(e) = client.init_db().await {
+            warn!(
+                log,
+                "failed to initialize single-node clickhouse database: {e}"
```

Comment:
Are we going to want to do the same for keepers and replicated servers?

Reply:
Yup! I'm going to do this very shortly.